lib/spaceship/tunes/tunes_client.rb

```diff
@@ -139,7 +149,8 @@ def send_login_request(user, password)
       request(:get, "https://itunesconnect.apple.com/WebObjects/iTunesConnect.woa")
 
       case response.status
-      when 403, 401
+      when 403
+        # FIXME: This is more invalid permissions than invalid credentials
```

Comment:
In theory 401 and 403 status codes are supposedly used for different purposes (https://en.wikipedia.org/wiki/HTTP_403#Difference_from_status_.22401_Unauthorized.22).

Maybe Apple isn't following this, but I somewhat doubt. I would like to investigate this further, because if Apple is using a 403 properly, then the error message spaceship returns could be more fine tuned. E.g. "you don't have permissions to access this resource", etc. E.g. if you don't have the proper role, or maybe if you haven't accepted the latest agreement, etc. This would be beneficial to users.

I'll create a new issue and reference this problem, and remove the FIXME from the PR.